config/initializers/money.rb

```diff
@@ -1,4 +1,8 @@
 ActiveSupport.on_load(:active_record) do
   ActiveRecord::Base.send(:include, ThreeScale::HasMoney)
 end
-ActionController::Base.helper(ThreeScale::MoneyHelper) if defined?(ActionController)
+
+Rails.application.config.to_prepare do
+  require 'three_scale/money'
```

Comment:
why is require needed here? Shouldn't it be auto-loaded?

Reply:
You're right. I removed this.

Reply:
After removing it, I found a problem: the required class included a required monkey patch. So I moved the monkey patch to it's own class out of the autoload path, and left the `ThreeScale::Money` in the autoload path. See 5c3101a838d9d7d4333b970eede969df74802fc0